packages/docs/content/components/drawer.mdx

```diff
@@ -50,6 +50,44 @@ A header and/or footer can have a "sticky" position if the following properties
   minHeight={300}
 />
 
+### Managing multiple drawers
+
+If you have multiple `<Drawer>`'s on a page, you can utilize the `<DrawerManager>` context provider along with the `useDrawerManager()` hook to manage the state of these drawers so that only one will remain open at a time. The hook provides functions to toggle state of the drawer via click, to close the drawer and to get the open status of the drawer. A minimal example of implementation:
+
+```js
+import { DrawerManager, useDrawerManager } from '@cmsgov/design-system';
+
+const ManagedDrawer = (props) => {
+  const { toggleClick, closeClick, isOpen } = useDrawerManager();
+
+  return (
+    <>
+      {isOpen && (
+        <Drawer onCloseClick={closeClick} {...props} />
+      )}
+      <Button
+        className="ds-c-drawer__toggle"
+        variation="ghost"
+        onClick={toggleClick}
+      >
+        Toggle {props.heading}
+      </Button>
+    </>
+  )
+}
+
+export default function App() {
+  <DrawerManager>
+    ... any content
+    <ManagedDrawer {...propsForDrawer1} />
+    <ManagedDrawer {...propsForDrawer2} />
+    <ManagedDrawer {...propsForDrawer3} />
+  </DrawerManager>
+}
+```
+
+Since this component utilizes React `Context`, any number of components/content items can be within the `DrawerManager` provider and will not be effected by it.
```

Comment:
Oh I see that you wrote that here. I think it would be appropriate to put this in the first paragraph, and I think we could add a link to React docs